blivet/devices/md.py

```diff
@@ -195,6 +195,13 @@ def level(self, value):
 
         self._level = level
 
+    @property
+    def sector_size(self):
+        if not self.exists:
+            return max(p.sector_size for p in self.parents)
```

Comment:
Is this enough, or does it need to be the least common multiple of the two? I imagine in most cases it will be the same, but will it always?

Reply:
I'm not sure if you can have a device with sector size that is not multiple of 512, but it might be possible. I've changed the code to calculate LCM of the sector sizes.